pkg/manifest/mantaray/marshal_test.go

```diff
@@ -201,3 +202,54 @@ func TestMarshal(t *testing.T) {
 	// 	}
 	// }
 }
+
+func Test_UnmarshalBinary(t *testing.T) {
+	decode := func(s string) []byte {
```

Comment:
Adding `t.Helper()` to the `decode` closure will help to accurately identify the problematic string.